New DevOps & Cloud Native Transformation page

znegrin opened the pull request:
As requested on the issue [#1404](https://github.com/src-d/backlog/issues/1404) of the Backlog, I added a page for the DevOps & Cloud Native Transformation solution.

Please notice that it is still missing the images.

![image](https://user-images.githubusercontent.com/14981468/60121412-4378a200-9783-11e9-8bd1-856e8aff2f2a.png)

znegrin commented:
> Body of the page LGTM, old header and footer though.

Once merged it will have the updated ones.

dpordomingo commented:
Already at staging https://landing-staging.srcd.run/solutions/devops-cloud-native-transformation/

znegrin commented:
I fixed the merge, reordered the layout to make it more compact and added the missing image. This page should be now completely ready:

![image](https://user-images.githubusercontent.com/14981468/60249429-21873880-98c5-11e9-95d3-505fc71a3235.png)
